packages/form/translations/de.yaml

```diff
@@ -51,3 +51,6 @@ caluma:
       uploadFailed: "Beim Hochladen ist ein Fehler aufgetreten."
       format: "{errorMsg}"
       table: "Mindestens eine Zeile der Tabelle wurde nicht korrekt ausgefüllt"
+      error: "Folgende Fragen sind noch nicht korrekt ausgefüllt:"
+      success: "Alle Fragen wurden korrekt ausgefüllt!"
+      loading: "Ihr Formular wird geprüft. Bitte haben Sie einen Moment Geduld."
```

Comment:
you can drop `success` and `loading`, since they're not used

Reply:
Done.